StringFormatter/Numeric.cs

```diff
@@ -216,6 +217,7 @@ unsafe static partial class Numeric {
                 NumberToCustomFormatString(formatter, ref number, specifier, culture);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Yeap, It's kind of the last thing I did with this, before submitting the PR, this was roughly a 10% improvement with RyuJit, and a 5% improvement for the legacy JIT.

if you feel strongly against it nonetheless I can remove, or separate into another PR